### ISSUE TITLE
[DISC] Improving Documentation

### ISSUE THREAD

**Issue by bhavnicksm** [chonkie-ai/chonkie#43]
Hey there!

Thank you for your support of 🦛 Chonkie✨!  I am extremely grateful to the community for the support and enthusiasm regarding Chonkie. 

As you might be aware, the Chonkie documentation at the moment is a simple DOCS.md file, which works out just fine as it stands, but as we expand Chonkie's feature set and capabilities to handle complex tasks and make it even faster, it would soon become hard to manage. 

Documentation for PyPI packages is generally pretty annoying and cumbersome. I hope to get something that's simple, straight-forward, and doesn't consume much time. 

A test for a good source of documentation is that:

*  It should not take more time to document a feature than it takes to code the feature
* Documentation inline is not fine if it makes the code look ugly and unreadible; I'd rather maintain a DOCS.md than an ugly codebase
* It should be open for easy contributions by community members (I'd be very happy to accept documentation PRs that make significant improvements in readability, btw) 

I am currently looking at options for this, and I hope I could receive some advice from the community on how to expand and scale the documentation better.

Thanks!

**Comment by mrmps** [comment]
I recommend mintlify

**Comment by mrmps** [comment]
Yup! It's super easy to get started, you just keep all your data in Markdown files. 

Mintlify's support is also extremely good, I had an issue and they responded pretty much immediately

**Comment by bhavnicksm** [comment]
@mrmps,

A bit late reply, but Mintlify is pretty cool! I've been trying to shift the documentation from `DOCS.md` to a mintlify docs recently...

I'll add the progress in the documentation shift in the thread! You can also check it out over on [bhavnicksm/chonkie-docs](https://github.com/bhavnicksm/chonkie-docs).

Of course, it's not up to date yet, but it's slowly getting there :)

Thanks for the rec!

**Comment by bhavnicksm** [comment]
Closing since it's a stale issue now, thanks!
